### PR TITLE
[GH-334] Check error message when genesis/last block too old.

### PR DESCRIPTION
**Pull request by ksaric**
https://github.com/input-output-hk/cardano-node/issues/334

The transformers were left alone since I see no reason to transform anything in here.
Wasn't tested with the local node.

### CODE REVIEW

File: cardano-node/src/Cardano/Tracing/ToObjectOrphans.hs [hunk 1]
```diff
@@ -781,6 +785,30 @@ instance ToObject (TraceLocalTxSubmissionServerEvent blk) where
...
-  toObject _verb _ =
-    mkObject [ "kind" .= String "TraceForgeEvent" ]
+instance ToObject (TraceForgeEvent blk) where
+  toObject _verb (TraceForgeEvent slotNo _) =
+    mkObject
+        [ "kind" .= String "TraceForgeEvent"
+        , "slot" .= toJSON (unSlotNo slotNo)
+        ]
+  toObject _verb (TraceCouldNotForge slotNo _) =
+    mkObject
```
Comment (CodiePP) [inline cardano-node/src/Cardano/Tracing/ToObjectOrphans.hs]: there is more information in the second parameter that could be traced here. `AnachronyFailure`

File: cardano-node/src/Cardano/Tracing/ToObjectOrphans.hs [hunk 2]
```diff
@@ -781,6 +785,30 @@ instance ToObject (TraceLocalTxSubmissionServerEvent blk) where
...
+        [ "kind" .= String "TraceAdoptedBlock"
+        , "slot" .= toJSON (unSlotNo slotNo)
+        ]
+  toObject _verb (TraceDidntAdoptBlock slotNo _) =
+    mkObject
+        [ "kind" .= String "TraceDidntAdoptBlock"
+        , "slot" .= toJSON (unSlotNo slotNo)
+        ]
+  toObject _verb (TraceForgedInvalidBlock slotNo _ _) =
+    mkObject
```
Comment (CodiePP) [inline cardano-node/src/Cardano/Tracing/ToObjectOrphans.hs]: there is more information in the second parameter that could be traced here. `InvalidBlockReason`

**Comment by ksaric** [comment]
Erm. Hydra failed?

**Comment by CodiePP** [comment]
bors try

**Comment by ksaric** [comment]
Chairman is failing.

**Pull request merged**